### PR TITLE
Add Script to Import DISA STIG to Policy Specific Content

### PR DESCRIPTION
**Pull request by Mab879**
#### Description:

This PR adds a script that takes the SRG Requirement, Fix text, check text, and Vulnerability discussion from the DISA STIG XML files and imports them to the Policy Specific Content (PSC) files.

#### Rationale:

Help prep for the next STIG.

#### Review Hints:
```
$ ./utils/import_disa_stig.py --product rhel9 --control stig_rhel9 shared/references/disa-stig-rhel9-v1r2-xccdf-manual.xml
```

Observe that under the rules used by the STIG the PSC content is updated.

### CODE REVIEW

File: utils/srg_utils/__init__.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+def get_rule_dir_json(path: str) -> dict:
+    with open(path, 'r') as f:
+        return json.load(f)
+
+
+def update_row(changed: str, current: str, rule_dir_json: dict, section: str) -> None:
+    if changed != current and changed:
+        replace_yaml_section(section, changed, rule_dir_json)
```
Comment (jan-cerny) [inline utils/srg_utils/__init__.py:37]:
```
[jcerny@fedora content]$  ./utils/import_disa_stig.py --product rhel9 --control stig_rhel9 shared/references/disa-stig-rhel9-v1r2-xccdf-manual.xml
Traceback (most recent call last):
  File "/home/jcerny/work/git/content/./utils/import_disa_stig.py", line 111, in <module>
    raise SystemExit(main())
                     ^^^^^^
  File "/home/jcerny/work/git/content/./utils/import_disa_stig.py", line 86, in main
    update_row(changed_fixtext, stig_srg_requirement, rule_obj, 'srg_requirement')
  File "/home/jcerny/work/git/content/utils/srg_utils/__init__.py", line 37, in update_row
    replace_yaml_section(section, changed, rule_dir_json)
    ^^^^^^^^^^^^^^^^^^^^
NameError: name 'replace_yaml_section' is not defined
```

File: utils/import_disa_stig.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        stig_fix_text = stig_rule['fixtext']
+        rule_fixtext = rule.policy_specific_content.get('fixtext')
+        if rule_fixtext != stig_fix_text:
+            changed_fixtext = fix_changed_text(stig_fix_text, changed_name)
+            update_row(changed_fixtext, rule_fixtext, rule_obj, 'fixtext')
+
+        stig_checktext = stig_rule['check']
+        rule_checktext = rule.policy_specific_content.get('checkfix')
+        if rule.checktext != stig_checktext:
```
Comment (jan-cerny) [inline utils/import_disa_stig.py:96]: rule_checktext ?

**Review by jan-cerny — CHANGES_REQUESTED** [review]

File: utils/import_disa_stig.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    control_name = args.control
+    stig_filename = args.input
+    build_config_yaml = args.build_config_yaml
+    changed_name = args.changed_name
+    env_yaml = _get_env_yaml(ssg_root, product, build_config_yaml)
+    controls = _get_controls(control_name, ssg_root, env_yaml)
+    rule_dir_json = get_rule_dir_json(args.json)
+    srgs = ssg.build_stig.parse_srgs(stig_filename)
+
+    for stig_id, stig_rule in srgs.items():
```
[the remark this answers is not in the document]
Comment (Mab879) [reply utils/import_disa_stig.py:68]: I have cleaned up the main method a bit, but moving the code the edits the file results in very ugly method signatures.

**Review by jan-cerny — APPROVED** [review]
I have executed the script and it updated the policy specific content as expected.

**Pull request merged**